lib/rules/max-nesting-depth/index.js

```diff
@@ -151,14 +184,6 @@ const rule = (primary, secondaryOptions) => {
 	}
 };
 
-/**
- * @param {string} selector
- * @returns {string | undefined}
- */
-function extractPseudoRule(selector) {
-	return selector.startsWith('&:') && selector[2] !== ':' ? selector.slice(2) : undefined;
-}
-
```

Comment:
[suggest] Could you please keep the original position of some functions to avoid potential conflicts with other PRs? In other words, I'd like to minimize diffs in the PR.